.github/workflows/pr.yaml

```diff
@@ -58,16 +66,9 @@ jobs:
             sleep 1
           done
           POD_NAME=$(kubectl get pod -o name | grep konkservice)
-          kubectl exec -it $POD_NAME -- kubectl get apiservice v1alpha1.test.infoblox.com
+          kubectl exec -it $POD_NAME -- kubectl describe apiservice v1alpha1.test.infoblox.com
+          kubectl exec -it $POD_NAME -- kubectl get apiservice v1alpha1.test.infoblox.com | grep True
```

Comment:
If we add this, the check on github will fail consistently, since we are not currently testing with a service as apart of the e2e test. (Unless there is plans to add one subsequently).

Reply:
Aren't we? apiserver gets deployed during the test.

Reply:
Oh that's right, we re-ordered that.